Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:latest
 
-RUN pip install metamist
```

Comment:
I don't think we want to drop this metamist install from the main production pipelines Dockerfile?